src/sass/rx/rx.scss

```diff
@@ -33,10 +33,12 @@
 }
 
 .rx-app {
-  background: $color-white;
-  padding: 3rem 1.5rem;
-  margin: auto;
-  max-width: $site-max-width;
+  margin-bottom: 3rem;
+
+  // TODO: Move to core?
+  .va-alert:first-child {
+    margin-top: 0;
```

Comment:
I see that there is a similar rule for `h1` in `_va.scss`, but not sure if we want to add it directly just yet.